test/Base.t.sol

```diff
@@ -245,6 +246,28 @@ abstract contract Base_Test is DeployOptimized, Events, Merkle, V2CoreAssertions
                                   MERKLE-STREAMER
     //////////////////////////////////////////////////////////////////////////*/
 
+    function createWithLockupLinear(
```

Comment:
This can't be placed in `Defaults` because it requires access to `lockupLinear` and `asset` variables which are defined in `Base_Test`.

```solidity
    function createWithLockupLinear(
        address admin,
        bytes32 merkleRoot,
        uint40 expiration
    )
        internal
        view
        returns (MerkleStreamer.CreateWithLockupLinear memory)
    {
        return MerkleStreamer.CreateWithLockupLinear({
            initialAdmin: admin,
            lockupLinear: lockupLinear,
            asset: asset,
            name: defaults.NAME_STRING(),
            merkleRoot: merkleRoot,
            expiration: expiration,
            streamDurations: defaults.durations(),
            cancelable: defaults.CANCELABLE(),
            transferable: defaults.TRANSFERABLE()
        });
    }
```

Reply:
See how i have handled it in my branch

Reply:
OK I will have a look at the other branch tomorrow first thing in the morning.